turbopack/crates/turbo-tasks-backend/src/backend/mod.rs

```diff
@@ -363,6 +374,11 @@ impl TurboTasksBackendInner {
                     task.add_new(CachedDataItem::AggregateRoot {
                         value: RootState::new(ActiveType::CachedActiveUntilClean, task_id),
                     });
+                    // A newly added AggregateRoot need to make sure to schedule the tasks
+                    task_ids_to_schedule = get_many!(task, AggregatedDirtyContainer { task } count if count.get(self.session_id) > 0 => task);
```

Comment:
This needs some sort of manual wrapping since rustfmt can't wrap macros like this
```suggestion
                    task_ids_to_schedule = get_many!(
                        task,
                        AggregatedDirtyContainer {
                            task
                        } count if count.get(self.session_id) > 0 => {
                            task
                        }
                    );

```